_literate/A-stacking.jl

```diff
@@ -96,10 +96,19 @@ y2 = predict(m2, X)
 
 yhat = 0.5*y1 + 0.5*y2
 
+surrogate = Deterministic()
+mach = machine(surrogate, Xs, ys; predict=yhat)
+
+fit!(yhat)
+ŷ(X[test, :])
+
```

Comment:
Something not right here. We have both a `yhat` and a `ŷ`. 

Also, I wouldn't construct the learning network machine until you export the network. This is a bit of distraction just now, and we're kind-of supposing the user has seen this before. Maybe you could just combine in one call, like this:

```julia
@from_network machine(Deterministic(), Xs, ys; predict=yhat) begin
     mutable struct MyAverageTwo
         regressor=model2
     end
 end
```

However, you decide if this is too terse and needs breaking down (I am used to it already 😸 ).